src/connection.rs

```diff
@@ -74,7 +74,63 @@ impl Connection {
     ///
     /// # Example
     /// ```
-    /// todo!()
+    /// use zbus::{fdo::DBusProxy, MessageType, MatchRule, Address};
+    /// use futures_lite::future::{block_on, race, yield_now};
+    /// use futures_lite::pin;
+    /// use futures_lite::StreamExt;
+    /// let receive_good_event = async {
+    ///    let connection = atspi::Connection::open().await.expect("Could not open a11y bus.");
```

Comment:
You can [hide parts of the example](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#hiding-portions-of-the-example) (eg. boiler plate) from from the reader by using '#' .